ETH/Overview.md

```diff
@@ -0,0 +1,42 @@
+# P14 ETH - Overview
+### 1 Definition
+
+Bitcoin and Ethereum are two major cryptocurrencies.
+
+| name              | definition                                          | unit    |
+| ----------------- | --------------------------------------------------- | ------- |
+| Bitcoin （BTC）   | decentralized currency （known as blockchain 1.0）  | Satoshi |
+| Ethereum  （ETH） | decentralized contract  （known as blockchain 2.0） | Wei     |
+
+### 2 Optimization of ETH compared to BTC
+
+Ethereum has made improvements to Bitcoin's operational issues in system design:
+
+- Shortened block generation time from 10 minutes to just over ten seconds and designed a consensus mechanism based on the **GHOST protocol**.
+
+- Changed mining puzzle: Bitcoin's is computationally intensive for hash value calculation with specialized mining equipment. Ethereum's is memory hard to resist ASIC chips. Future change will replace proof of work with proof of stake.
+
+In addition, Ethereum supports **smart contracts**.
+
+### 3 What is a decentralized contract? 
+
+- After Bitcoin's success, people wonder what else can be decentralized.The emergence of Ethereum has added the characteristic of decentralized contracts. 
```

Comment:
The following two line seems to be "How decentralized contract is inspired"? Not "hat is decentralized contract".

I think note should be concise and structured.